src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -221,6 +247,15 @@ private void CheckIfExportIsEnabled()
             }
         }
 
+        private void ValidateForAnonymizedExport(string containerName, string anonymizationConfigLocation, string anonymizationConfigFileETag)
+        {
+            if (!string.IsNullOrWhiteSpace(anonymizationConfigLocation) || !string.IsNullOrWhiteSpace(anonymizationConfigFileETag))
+            {
+                CheckIfAnonymizedExportIsEnabled();
+                CheckContainerNameAndConfigLocationForAnonymizedExport(containerName, anonymizationConfigLocation);
+            }
+        }
+
         private void CheckIfAnonymizedExportIsEnabled()
```

Comment:
>CheckIfAnonymizedExportIsEnabled [](start = 21, length = 32)

nit: you can probably put this logic into `ValidateForAnonymizedExpor` and have less functions in a class

Reply:
Sure, moved this into the ValidateForAnonymizedExport.